### PR TITLE
Add additional tests for Core config and functions files

### PR DESCRIPTION
**Pull request by rwhite27**

### CODE REVIEW

File: tests/unit/AppsCest.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+namespace Canvas\Tests\unit;
+
+use Canvas\Providers\DatabaseProvider;
+use Phalcon\Di\FactoryDefault;
+use UnitTester;
+use Canvas\Models\Apps;
+
+class AppsCest
+{
+    public function setDatabaseProvider(UnitTester $I){
```
Comment (kaioken) [inline tests/unit/AppsCest.php]: @rwhite27 coding style

**Comment by kaioken** [comment]
@rwhite27 you have a conflict and we need to add the migration to this

**Comment by kaioken** [comment]
@rwhite27 need to fix conflicts

File: src/Core/config.php [hunk 2]
```diff
@@ -143,7 +143,7 @@
         'queue' => envValue('PUSHER_QUEUE'),
     ],
     'stripe' => [
-        'secret' => getenv('STRIPE_SECRET'),
```
Comment (kaioken) [inline src/Core/config.php]: @rwhite27 doesnt this affect the api?

Comment (rwhite27) [reply src/Core/config.php]: yes. Everything actually.

**Pull request merged**